helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -176,7 +207,13 @@ def create(self, vals_list):
                 )
                 if channel_email_id:
                     vals["channel_id"] = channel_email_id.id
-        return super().create(vals_list)
+        res = super().create(vals_list)
+        created_ticket_template_name = self._get_created_ticket_template_name()
+        if created_ticket_template_name:
+            for rec in res:
+                if rec.partner_id or rec.partner_email:
+                    rec._send_mail_to_partner(created_ticket_template_name)
+        return res
```

Comment:
**question**: if you have an email template set on the stage, and this global email template on ticket creation, do both emails get sent at the same time?